usaspending_api/accounts/urls_federal_accounts_v2.py

```diff
@@ -6,5 +6,5 @@
 object_class_federal_accounts = views.ObjectClassFederalAccountsViewSet.as_view()
 
 urlpatterns = [
-    url(r'^object_class/(?P<pk>[0-9]+)/$', object_class_federal_accounts)
+    url(r'^available_object_classes/(?P<pk>[0-9]+)$', object_class_federal_accounts)
```

Comment:
After looking at the gist, is it possible to alter the endpoint to be something like `/api/v2/federal_account/:federal_account_id/available_object_classes/`? This clarifies a bit with what's going on: We're getting a specific federal account by the ID and then getting the available object classes for that federal account.